changelog/seed/dog-breed.csv

```diff
@@ -1,14 +1,5 @@
 breed, active
```

Comment:
The breeds I added are real ones from the Access DB, and if removed will mean the Access DB import will fail on all records. So I'd like to keep these in if poss. We could potentially remove N/A but probably need to keep 'Withdrawn'